.github/workflows/build-lint-test.yaml

```diff
@@ -1,13 +1,12 @@
 name: build-lint-test
 
-on:
-  pull_request:
+on: [push, workflow_dispatch]
 
 env:
   LIBCURL_RELEASE: 'LATEST'
 
 concurrency:
-  group: ${{ github.head_ref }}
+  group: ${{ github.ref }}
```

Comment:
This and the above change should probably be reverted, as I did those for ease of testing.

Reply:
reverted